### PR TITLE
Bug 1874252: pinning image to rhcos 4.5 for step registry proxy config

### PR DESCRIPTION
**Pull request by ewolinetz**
Pinning to RHCOS 4.5 due to 4.6's pivot to using ignition v3.1.0 (which doesn't work with 4.5 and older) re: https://github.com/openshift/installer/pull/3871

Addresses https://bugzilla.redhat.com/show_bug.cgi?id=1874252

### CODE REVIEW

**Comment by mfojtik** [comment]
/lgtm

The job at least run, so it is a progress :-)

**Comment by mfojtik** [comment]
/retest

I would merge this regardless of what the rehearse say?

**Comment by ewolinetz** [comment]
@mfojtik rehearse isn't a blocker for this to merge. the cluster is able to install correctly with this change, if a test fails it would be due to a component given this passes on 4.5

**Comment by mfojtik** [comment]
@ewolinetz can you summarize why we need 4.5 in 4.6 test? is this CI only or will any customer installing 4.6 behind proxy have to go over this problem?

**Comment by wking** [comment]
/hold cancel

This is just about our CI setup, it has nothing to do with in-product proxy support.

**Comment by bparees** [comment]
/hold 
rehearsal still failed

**Comment by wking** [comment]
Looking at [the previous, failed proxy rehearsal][1], I'm not clear on why [we still failed to SSH into the proxy instance][2].

[1]: https://prow.ci.openshift.org/view/gs/origin-ci-test/pr-logs/pull/openshift_release/11453/rehearse-11453-pull-ci-openshift-installer-master-e2e-aws-proxy/1300799279656341504
[2]: https://storage.googleapis.com/origin-ci-test/pr-logs/pull/openshift_release/11453/rehearse-11453-pull-ci-openshift-installer-master-e2e-aws-proxy/1300799279656341504/artifacts/e2e-aws-proxy/gather-proxy/container-logs/test.log

**Comment by bparees** [comment]
/hold cancel

moves the ball forward, but more work is needed
